lib/imagec/imagec.go

```diff
@@ -158,29 +155,21 @@ func init() {
 }
 
 // ParseReference parses the -reference parameter and populate options struct
-func (ic *ImageC) ParseReference() error {
-	// Validate and parse reference name
-	ref, err := reference.ParseNamed(ic.Reference)
-	if err != nil {
-		log.Warn("Error while parsing reference %s: %#v", ic.Reference, err)
-		return err
-	}
-
-	ic.Tag = reference.DefaultTag
-	if !reference.IsNameOnly(ref) {
-		if tagged, ok := ref.(reference.NamedTagged); ok {
+func (ic *ImageC) ParseReference() {
+	if reference.IsNameOnly(ic.Reference) {
+		ic.Tag = reference.DefaultTag
+	} else {
+		if tagged, isTagged := ic.Reference.(reference.NamedTagged); isTagged {
```

Comment:
I see. Here, we want to set `ic.Tag` if the type-assert is successful, so we may want to keep this as-is.